packages/next/src/client/components/react-dev-overlay/internal/helpers/hydration-error-info.ts

```diff
@@ -51,14 +74,18 @@ export const getReactHydrationDiffSegments = (msg: NullableText) => {
 export function storeHydrationErrorStateFromConsoleArgs(...args: any[]) {
   const [msg, serverContent, clientContent, componentStack] = args
   if (isKnownHydrationWarning(msg)) {
-    hydrationErrorState.warning = [
-      // remove the last %s from the message
-      msg,
-      serverContent,
-      clientContent,
-    ]
+    hydrationErrorState.warning = [msg, serverContent, clientContent]
     hydrationErrorState.componentStack = componentStack
     hydrationErrorState.serverContent = serverContent
     hydrationErrorState.clientContent = clientContent
+
+    return [
+      ...args,
+      // We tack on the hydration error message to the console.error message so that
+      // it matches the error we display in the redbox overlay
+      `\nSee more info here: https://nextjs.org/docs/messages/react-hydration-error`,
```

Comment:
Supports behavior that was lost from:
- https://github.com/vercel/next.js/pull/67011